app/views/projects/show.html.erb

```diff
@@ -0,0 +1,4 @@
+<h3> <%= @project.name %> </h3>
+<p>Material: <%= @project.material %> </p>
+<p>Challenge Theme: <%= @challenge.theme %> </p>
+<p>Number of contestants: <%= @project.contestants.count %> </p>
```

Comment:
I'm sure Khoa talked about this so you probably know, but this method of .count, you'd want to make a method in the Project model that's called something like `contestant_count` and this logic would be encapsulated there. Activerecord methods should be confined to models.